src/components/Editor/CallSites.js

```diff
@@ -99,9 +99,12 @@ class CallSites extends Component {
       return;
     }
 
+    const { sourceId } = selectedLocation;
     const { line, column } = getTokenLocation(editor.codeMirror, target);
-
-    this.toggleBreakpoint(line + 1, column - 2);
+    this.toggleBreakpoint(
```

Comment:
i suppose we can't use a similar `toSourceLocation` method here because of the strange column offset...